code/modules/projectiles/projectile/beams.dm

```diff
@@ -138,9 +138,8 @@
 	name = "stun beam"
 	icon_state = "stun"
 	fire_sound = 'sound/weapons/Taser.ogg'
-	check_armour = "energy"
 	sharp = 0 //not a laser
-	agony = 40
```

Comment:
this is a bit larger than we'd usually buff in a single step, go with 45 for now